Random fixes

ljsinclair opened the pull request:
Minor fixes to:
* remove backticks from table headers
* first heading to make H1 rather than H2
* fixes to malformed tables and punctuation in headings
* fix position of before you begin in a couple of SQL pages
* fixed 6 duplications of the same table create statements on 3 pages with a single includes

gthrone25 commented on _includes/com-ingest/com-ingest-flag-sql-connect-string.md:
```diff
@@ -1,6 +1,6 @@
 ### SQL connection strings
 
-| `--driver` | example `--connection-string` | Further information |
+| driver | example connection-string | Further information |
```
This is a command line flag, right? doesn't that mean the `--` is needed?

ljsinclair replied:
I'm going to need to revisit.
The problem with putting `--` without the backticks is the system auto-changes these to an M-dash.
And the reason I changed it is because backticks in headers makes the text impossible to read >.<
Theoretically the quickest fix would be to muck around with CSS. But that's not my final word.

gthrone25 replied:
Then maybe just don't make it code here? It's better to have it correctly imo than as a code block. Not worth spending a lot of time on :)

ljsinclair replied:
It works now I've made change to CSS and tested.

gthrone25 commented on docs/sql-guide/functions/function-totimestamp.md:
```diff
@@ -57,17 +58,20 @@ insert into demo(_id, ts)
-To convert integer date-time values with non-standard granularity TOTIMESTAMP() can be used with appropriate `timeunit` parameter that matches the granularity of the input. 
+Result:
+
+| _id | ts |
```
I'm actually a little worried about doing this because the majority of examples are not returned as tables, but as the natural output of fbsql. Converting results into tables can take a lot of time, especially for result sets with many columns. Thoughts? Were you planning to do this for all examples?

ljsinclair replied:
Natural output wins.

So would it look like the original content I turned into a table? Or something else?

gthrone25 replied:
Ya the original content I believe. the majority of our sql function examples just has the natural output in the code block: https://docs.featurebase.com/docs/sql-guide/functions/function-prefix/

gthrone25 reviewed (commented):
Looks good but I put two comments in there to discuss!

ljsinclair commented on _sass/color_schemes/featurebase.scss:
```diff
@@ -17,3 +17,11 @@ $link-color: $purple-000;
+// Syntax highlighting with dark purple text
+// makes code block legible in table headings
+
+.language-plaintext, .highlighter-rouge {
```
This is the fix so code blocks are legible in table headers.

gthrone25 replied:
Up to you to make this but PLEASE do validation that this change doesn't mess up other formatting elsewhere. I've seen that before, and I'd rather just not have codeblock in table headers than break other formatting as neither of us know css well. Change up to you but validation throughout docs (visual inspection) is needed

ljsinclair commented on docs/sql-guide/data-types/data-type-timestamp.md:
```diff
@@ -35,6 +36,39 @@ TIMESTAMP [TIMEUNIT {value}]
 ## Examples
 
-### CREATE TABLE with all data types
+### Implicitly convert integer to timestamp
```
content moved here from TOTIMESTAMP() because it is explicitly about timestamp and doesn't feature anything with TOTIMESTAMP()

ljsinclair commented on docs/sql-guide/functions/function-setcontains.md:
````diff
@@ -12,15 +12,15 @@ SETCONTAINS() tests membership of a value within a set expression.
 
 ```
-setcontains(set, value)
+setcontains({set, value})
````
added brackets because these are both required and so the syntax was wrong

gthrone25 replied:
We haven't done brackets for any of the sql functions to my knowledge. The only place that exists is for data types.  We say required or not  in the args. We need standards but let's not solve this right now

ljsinclair commented on docs/sql-guide/functions/function-setcontains.md:
```diff
@@ -12,15 +12,15 @@ SETCONTAINS() tests membership of a value within a set expression.
-|---|---|---|---|
-| `set` | The set in which value is being tested for membership. | `stringset` or `idset` |
-| `value` | The value to test membership for in the set. `value` must be assignment compatible with the element type of the set. |
+| Argument | Description | Required | Data type |
```
added `Required` column to back-up the change to the syntax

gthrone25 replied:
Ya we need standards on this stuff. Data type in a different spot here: https://docs.featurebase.com/docs/sql-guide/functions/function-prefix/. Further information on some. Let's not solve this hear. This is stuff on the top of my list post re-writes.

gthrone25 reviewed (commented):
@ljsinclair please make the changes in comments and let's get this closed out and not keep adding to it. SQL standards is a bigger item that I want to get to after rewrites

ljsinclair commented:
> @ljsinclair please make the changes in comments and let's get this closed out and not keep adding to it. SQL standards is a bigger item that I want to get to after rewrites

- confirmed the SCSS change doesn't break anything
- reverted TOTIMESTAMP to earlier version as it'll be revisited later
- removed curly brackets where directed as we're revisiting

gthrone25 reviewed (approved):
Looks good!
